Casks/chromium.rb

```diff
@@ -1,14 +1,18 @@
 cask "chromium" do
-  version "838550"
+  version "839126"
   sha256 "53dc415319d467fe69dde10867dbff37247b5a9f4fb7f84e8332913d20174b49"
 
   url "https://commondatastorage.googleapis.com/chromium-browser-snapshots/Mac/#{version}/chrome-mac.zip",
       verified: "commondatastorage.googleapis.com/chromium-browser-snapshots/Mac/"
-  appcast "https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Mac%2FLAST_CHANGE?alt=media"
   name "Chromium"
   desc "Free and open-source web browser"
   homepage "https://www.chromium.org/Home"
 
+  livecheck do
+    url "https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Mac%2FLAST_CHANGE?alt=media"
+    regex(/\d+(?:\.\d+)*/)
```

Comment:
```suggestion
    regex(/v?(\d+(?:\.\d+)*)/i)
```

We use a capture group around the version in regexes.